SeedJobs/bmr-react-scripts
# Add watch functionality

## Testing steps

- Use app-grid, follow steps in https://github.com/SeedJobs/app-grid/pull/284 pr to setup locally (uses this branch)
- Run `beam app start grid`
- Visit `app.beamery.local` (or log in via `admin.beamery.local`)
   - Visit `https://app.beamery.local/#/crm/candidates`
   - Make a change in app-grid src (e.g. add an unused const to a component file)
   - See page reload in app.beamery.local
